lib/ansible/module_utils/network/junos/facts/facts.py

```diff
@@ -24,6 +25,7 @@
 FACT_RESOURCE_SUBSETS = dict(
     interfaces=InterfacesFacts,
     lag_interfaces=Lag_interfacesFacts,
+    l2_interfaces=L2_interfacesFacts,
```

Comment:
this would need to be rebased as well, + the imports

Reply:
Fixed